### PR TITLE
sort car urls

### PR DESCRIPTION
**Pull request by anjor**
for https://github.com/rpcpool/yellowstone-faithful/issues/169

### CODE REVIEW

File: cmd-car-split.go [hunk 1]
```diff
@@ -603,3 +605,113 @@ func SortCarFiles(carFiles []string) ([]string, error) {
...
+
+	// Read the partial content
+	partialContent, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return 0, fmt.Errorf("failed to read partial content: %w", err)
+	}
+
+	readCloser := io.NopCloser(bytes.NewReader(partialContent))
+
+	cr, err := carreader.New(readCloser)
```
Comment (gagliardetto) [inline cmd-car-split.go:691]: car header is at the beginning of the car file

**Review by gagliardetto — CHANGES_REQUESTED** [review]

File: cmd-car-split.go (hunk 1 above)
Comment (anjor) [reply cmd-car-split.go:691]: oh haha, of course.

Comment (anjor) [reply cmd-car-split.go]: @gagliardetto fixed.

**Review by gagliardetto — CHANGES_REQUESTED** [review]
please add a test with unsorted cars (with a test http server, test split car files, etc)

**Review by gagliardetto — APPROVED** [review]

**Pull request merged**